src/mixer/playermanager.cpp

```diff
@@ -361,67 +358,73 @@ void PlayerManager::addConfiguredDecks() {
 
 void PlayerManager::addDeckInner() {
     // Do not lock m_mutex here.
-    QString group = groupForDeck(m_decks.count());
-    VERIFY_OR_DEBUG_ASSERT(!m_players.contains(group)) {
+    ChannelHandleAndGroup handle_group =
+            m_pEngine->registerChannelGroup(groupForDeck(m_decks.count()));
+    VERIFY_OR_DEBUG_ASSERT(!m_players.contains(handle_group.handle())) {
         return;
     }
 
-    int number = m_decks.count() + 1;
+    int index = m_decks.count() + 1;
 
-    EngineChannel::ChannelOrientation orientation = EngineChannel::LEFT;
-    if (number % 2 == 0) {
-        orientation = EngineChannel::RIGHT;
-    }
-
-    Deck* pDeck = new Deck(this, m_pConfig, m_pEngine, m_pEffectsManager,
-            m_pVisualsManager, orientation, group);
+    Deck* pDeck = new Deck(this,
+            m_pConfig,
+            m_pEngine,
+            m_pEffectsManager,
+            m_pVisualsManager,
+            index % 2 == 0 ? EngineChannel::RIGHT : EngineChannel::LEFT,
+            handle_group);
     connect(pDeck->getEngineDeck(),
             &EngineDeck::noPassthroughInputConfigured,
             this,
             &PlayerManager::noDeckPassthroughInputConfigured);
-    connect(pDeck, SIGNAL(noVinylControlInputConfigured()),
-            this, SIGNAL(noVinylControlInputConfigured()));
+    connect(pDeck,
+            SIGNAL(noVinylControlInputConfigured()),
+            this,
+            SIGNAL(noVinylControlInputConfigured()));
```

Comment:
Please update to use Qt5 function pointer signal-slot syntax if you change these.

Reply:
I didn't change these, this was clang-format ^^

Reply:
Why? The line is unchanged so git clang-fromat should not touch it.

Reply:
Yeah hm sometimes happens to me, probably my IDE being a bit overeager

Reply:
If you can't stop you IDE doing this, you kind of adopt these lines. When touching them they need they need to be complient to our current rules and you will receive review comments. 

Revert or fix?

Reply:
Already adjusted it, just didn't push yet :)